sway/commands.c

```diff
@@ -241,6 +246,14 @@ static bool cmd_floating(struct sway_config *config, int argc, char **argv) {
 	return true;
 }
 
+static bool cmd_floating_mod(struct sway_config *config, int argc, char **argv) {
+	if (!checkarg(argc, "floating_modifier", EXPECTED_EQUAL_TO, 1)) {
+		return false;
+	}
+	config->floating_mod = xkb_keysym_from_name(argv[0], XKB_KEYSYM_CASE_INSENSITIVE);
```

Comment:
Actually it shouldn't, I changed some code a while back that made is so that varsubs are done on all parts of every command.